docs/BR.md

```diff
@@ -826,7 +826,7 @@ The file containing the Request Token or Random Number:
 If the CA follows redirects, the following apply:
 
 1. Redirects MUST be initiated at the HTTP protocol layer.
-   a. For validations performed on or after July 1, 2021, redirects MUST be the result of a 301, 302, or  307 HTTP status code response, as defined in [RFC 7231, Section 6.4](https://tools.ietf.org/html/rfc7231#section-6.4), or a 308 HTTP status code response, as defined in [RFC 7538, Section 3](https://tools.ietf.org/html/rfc7538#section-3). Redirects MUST be to the final value as defined in [RFC 7231, Section 7.1.2](https://tools.ietf.org/html/rfc7231#section-7.1.2) specified in the Location HTTP response header.
+   a. For validations performed on or after July 1, 2021, redirects MUST be the result of a 301, 302, or 307 HTTP status code response, as defined in [RFC 7231, Section 6.4](https://tools.ietf.org/html/rfc7231#section-6.4), or a 308 HTTP status code response, as defined in [RFC 7538, Section 3](https://tools.ietf.org/html/rfc7538#section-3). Redirects MUST be to the final value of the Location HTTP response header, as defined in [RFC 7231, Section 7.1.2](https://tools.ietf.org/html/rfc7231#section-7.1.2).
```

Comment:
@sleevi I believe this alteration makes the intended meaning of the point a little less clear. What does "final value of" mean in this context? I'm assuming that it's something along the lines of "Redirects MUST follow the Location HTTP response header and the Location HTTP response header MUST point to the page containing the random value"?

Reply:
Sorry, meant to tag @CBonnell as well.

Reply:
A redirect can be to a relative URI, at which point, the HTTP client needs to resolve that relative URI to the document URI.

The term final value is documented in https://tools.ietf.org/html/rfc7231#section-7.1.2 which is referred to by these status codes. It probably didn’t show up in a Find-in-Page for you because of the white space of the RFC, which tripped me up as well in the past.

Reply:
Thanks for recognizing my lack of searching skill, lol! That does help clarify the meaning.